src/MobileDevice/widget/MobileDevice.xml

```diff
@@ -0,0 +1,66 @@
+    <properties>
+        <property key="deviceEntity" type="entity" required="false">
```

Comment:
Yes, you need it, but when the widget requires a context it always expect the MF to have an input equal to the context. So no need to specify it in the xml.